chart/templates/webserver/webserver-deployment.yaml

```diff
@@ -171,6 +175,7 @@ spec:
           {{- end }}
           resources:
 {{ toYaml .Values.webserver.resources | indent 12 }}
+          securityContext: {{ or .Values.webserver.securityContexts.container .Values.securityContexts.container }}
```

Comment:
In my local testing, this adds a duplicate `securityContext` key

Reply:
fused the duplicate keys into one https://github.com/apache/airflow/pull/24588/files#diff-42f2a460a872212b923b5d7591c552a22816d768478a70d6943103221e08f11fR169